src/_base/harness/config/functions.yml

```diff
@@ -114,6 +114,30 @@ function('get_docker_external_networks'): |
   }
   = join(" ", $externalNetworks);
 
+function('docker_service_images'): |
```

Comment:
I think we need to account for the `build: path/to/context` too

Reply:
no, `docker-compose config` converts it to build.context

Reply:
Sweet :)